Basic/Invaders/Assets/Scripts/SceneTransitionHandler.cs

```diff
@@ -106,18 +113,12 @@ public void SwitchScene(string scenename)
         }
     }
 
-    void OnLoadComplete(ulong clientId, string sceneName, LoadSceneMode loadSceneMode)
+    private void OnLoadComplete(ulong clientId, string sceneName, LoadSceneMode loadSceneMode)
```

Comment:
Nitpick: private.

Reply:
I added it here to be consistent with the rest of the project